apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -569,6 +573,34 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
           )}
         />
       )}
+      {hasValidLicense && (
```

Comment:
We now only allow these options for enterprise customers. We'll still keep both options available but will replace once we allow users to set granular options for emails
![CleanShot 2024-07-25 at 15 18 50@2x](https://github.com/user-attachments/assets/e2d565d8-5a89-45d1-8140-8debdee3132f)